routes/web.php

```diff
@@ -11,6 +11,17 @@
 |
 */
 
+// Developer/testing helper routes
+
+if (env('APP_DEBUG')) {
```

Comment:
This should be retrieved from config so it can be cached using config caching

Reply:
Fixed in c8b76fa